.github/release_plan.md

```diff
@@ -57,12 +57,11 @@
 
 ## Release
 
--   [ ] Publish the release via Azure DevOps
-    -   [ ] Make sure [CI](https://github.com/Microsoft/vscode-jupyter/blob/main/CONTRIBUTING.md) is passing. Try a re-run on any failing CI test stages. If tests still won't pass contact the owning team.
-    -   [ ] On Actions tab of GitHub, ensure a workflow has automatically run for the workflow `Release VSIX`.
-    -   [ ] Download the [Release VSIX](https://pvsc.blob.core.windows.net/extension-builds-jupyter/ms-toolsai-jupyter-release.vsix) & Make sure no extraneous files are being included in the `.vsix` file (make sure to check for hidden files)
-    -   [ ] Go into the `Publish Extension` workflow and manually run this workflow to publish the extension to the marketplace.
-    -   [ ] From a VSCode instance uninstall the python extension. After the publish see if the new version is available from the extensions tab. Download it and quick sanity check to make sure the extension loads.
+-   [ ] Publish the release manually
```

Comment:
This is a manual step for now (GitHub actions support this, but we haven't enabled it for now)
I guess we can add support later.